### ISSUE TITLE
Create Your Own Wake Word tutorial no longer works

### ISSUE THREAD

**Issue by codefaux** [home-assistant/home-assistant.github.io#35782]
### Feedback

This tutorial no longer works, as the Python notebook does not execute properly.
![image](https://github.com/user-attachments/assets/d536400c-9830-4ec6-b684-e1245047aea6)

Error occurs during final notebook step aka Step 3. Train the Model. Prior steps complete with no error. Voice samples generate properly.

Text of error:

```
TypeError: Descriptors cannot be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
```

### URL

https://www.home-assistant.io/voice_control/create_wake_word/

### Version

2024.11.2

**Comment by mute2014** [comment]
tried it today, same error.

**Comment by codefaux** [comment]
Lots of eyes likely on the notebook right now -- NetworkChuck just did a video series on HASS voice stuff and his setup dictates using this notebook in the process.

**Comment by codefaux** [comment]
Yeah apparently someone noticed and fixed it without anyone noticing the issue for a few months. Congrats.
